ui/userguide/UserGuideBody.jsx

```diff
@@ -0,0 +1,209 @@
+import React from 'react';
+
+import ShortcutTable from '../shared/ShortcutTable';
+
+import logviewerIconHelp from '../img/logviewerIconHelp.svg';
+
+const possiblesNotation = [
+  {
+    text: '+n',
+    classes: 'btn-green ug-btn-comment',
+    explanation: 'Collapsed job count',
+  },
+  {
+    text: 'Th()',
+    classes: 'btn-dkgray ug-btn-comment',
+    explanation: 'Wrapped job group',
+  },
+  {
+    classes: 'btn-orange-classified autoclassified ug-btn-comment',
+    explanation: 'Hollow asterisk, auto-classified',
+  },
+  {
+    classes: 'btn-orange-classified ug-btn-comment',
+    explanation: 'Asterisk, classified',
+  },
+  {
+    classes: 'btn-ltgray ug-btn-bg',
+    explanation: 'Light gray, pending',
+  },
+  {
+    classes: 'btn-dkgray ug-btn-bg',
+    explanation: 'Gray, running',
+  },
+  {
+    classes: 'btn-green ug-btn-bg',
+    explanation: 'Green, success',
+  },
+  {
+    classes: 'btn-orange ug-btn-orange',
+    explanation: 'Orange, tests failed',
+  },
+  {
+    classes: 'btn-purple ug-btn-purple',
+    explanation: 'Purple, infrastructure exception',
+  },
+  {
+    classes: 'btn-red ug-btn-red',
+    explanation: 'Red, build error',
+  },
+  {
+    classes: 'btn-dkblue ug-btn-bg',
+    explanation: 'Dark blue, build restarted',
+  },
+  {
+    classes: 'btn-pink ug-btn-bg',
+    explanation: 'Pink, build cancelled',
+  },
+  {
+    classes: 'btn-yellow ug-btn-yellow',
+    explanation: 'Yellow, unknown',
+  },
+  {
+    classes: 'btn-ltblue ug-btn ug-btn-bg',
+    explanation: 'Light blue, superseded',
+  },
+];
+
+class UserGuideBody extends React.Component {
+  componentWillMount() {
+    // Used for dynamic startdate, enddate param examples
+    const d = new Date();
+    const y = d.getFullYear();
+    const m = d.getMonth() + 1;
+    const sd = d.getDate() - 2;
+    const ed = d.getDate();
+
+    this.endDate = `${y}-${m}-${ed}`;
+    this.startDate = `${y}-${m}-${sd}`;
+  }
+
+  render() {
+    const notations = possiblesNotation.map((notation) => {
```

Comment:
Also with a good practice, we not should to use iterations inside render method, I will go change to this;
`const Notations = notations => notations.map(notation => ...)`